### PR TITLE
Added DNS Spoof chaos

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

This PR:
- Adds charts for dns spoof chaos
- Renames `pod-dns-chaos` to `pod-dns-error`

### CODE REVIEW

File: charts/generic/pod-dns-error/pod-dns-chaos.chartserviceversion.yaml [hunk 1]
```diff
@@ -1,37 +1,37 @@
 apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
-  name: pod-dns-chaos 
+  name: pod-dns-error
```
Comment (ispeakc0de) [inline charts/generic/pod-dns-error/pod-dns-chaos.chartserviceversion.yaml]: Can we rename this file?

File: charts/generic/pod-dns-spoof/experiment.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    injects dns failure in target pod container
```
Comment (ispeakc0de) [inline charts/generic/pod-dns-spoof/experiment.yaml]: can we update the description to more specific?

File: charts/generic/pod-dns-spoof/pod-dns-chaos.chartserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
```
Comment (ispeakc0de) [inline charts/generic/pod-dns-spoof/pod-dns-chaos.chartserviceversion.yaml]: Can we rename this file?

File: charts/generic/generic.chartserviceversion.yaml [hunk 4]
```diff
@@ -35,8 +35,9 @@ spec:
     - node-io-stress
     - pod-io-stress
     - node-restart
-    - pod-dns-chaos
-    
+    - pod-dns-error
```
Comment (ispeakc0de) [inline charts/generic/generic.chartserviceversion.yaml:38]: Can we add the icons for both experiments?

**Review by ispeakc0de — DISMISSED** [review]

File: charts/generic/pod-dns-spoof/experiment.yaml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        value: "/var/run/docker.sock"
+
+      ## it defines the sequence of chaos execution for multiple target pods
+      ## supported values: serial, parallel
+      - name: SEQUENCE
+        value: "parallel"
+
+      # map of the target hostnames eg. '{"abc.com":"spoofabc.com"}' . If empty no queries will be spoofed
+      - name: SPOOF_MAP
+        value: '{"abc.com":"spoofabc.com"}'
```
Comment (ksatchit) [inline charts/generic/pod-dns-spoof/experiment.yaml]: better to keep this empty and make it a mandatory var in docs? We anyways have the comment in the source/repo.

**Review by ksatchit — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

File: charts/generic/pod-dns-error/experiment.yaml [hunk 6]
```diff
@@ -1,12 +1,12 @@
 apiVersion: litmuschaos.io/v1alpha1
 description:
   message: |
-    injects dns failure in target pod container
+    Pod DNS Error injects dns failure/error in target pod containersr
```
Comment (uditgaurav) [inline charts/generic/pod-dns-error/experiment.yaml:4]:
```suggestion
    Pod DNS Error injects dns failure/error in target pod containers
```

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**